tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
- "module": "commonjs // Enable require and module.exports
- "esModuleInterop": true // // Enables interoperability allowing import 
- "skipLibCheck": true // Skips type checking in library definition files (`.d.ts` files)
- "forceConsistentCasingInFileNames": true // Enforces consistent casing in filenames